cmd/serve.go

```diff
@@ -137,20 +140,14 @@ func serve(ctx context.Context) error {
 	}
 
 	// jwt auth middleware
-	if !serveDevMode {
-		if authconfig, err := echojwtx.AuthConfigFromViper(viper.GetViper()); err != nil {
+	if viper.GetBool("oidc.enabled") {
```

Comment:
isn't oidc enabled by default? Wouldn't we still want to be able to turn off a bunch of things via that `--dev` flag?